vue-app/src/components/WalletModal.vue

```diff
@@ -85,9 +82,6 @@ export default class WalletModal extends Vue {
   async loginUser(user: User): Promise<void> {
     this.$emit('connected')
 
-    const url = await getProfileImageUrl(user.walletAddress)
-    this.updateProfileImage(url)
```

Comment:
Removed this "task" from here (there was no reason to let the `loginUser` take care of that).

Now the consumers fetch the image when the user is connected. We are doing this in the `WalletWidget` by watching the `currentUser` object.